### DISCUSSION TITLE
Plot. lines - days

### DISCUSSION THREAD

**Discussion by PaulaIzzo** [observablehq/plot#2253] [Q&A] [answered]
Hello, I am learning to use observablehq and when I make this graph, the dates on the x axis are out of phase. Thanks.
```js
lin_cal = Plot.plot({
  title: "Línea de Tiempo de Calorías Diarias",
  x: {
    label: "Día",
    type: "time", 
    ticks: 5, 
    tickFormat: d3.timeFormat("%b %d"), 
  },
  y: { 
    label: "Calorías", 
    grid: true 
  },
  marks: [
    Plot.line(caloriasPorDia, { x: "Día", y: "Calorías", strokeWidth: 3 }),
    Plot.dot(caloriasPorDia, { x: "Día", y: "Calorías", tip: true }),
    Plot.ruleY([1200], { stroke: null })
  ],
  marginBottom: 70,
});
```
![image](https://github.com/user-attachments/assets/b1766a2a-6abc-47e8-abdd-3caecf8ae079)

**Comment by Fil** [comment, marked as answer]
It's hard to tell without seeing the data, but maybe try `type: "utc", ` instead of `type: "time",`? the "time" type works in local time, including time zone.

**Reply by PaulaIzzo** [reply]
When I indicate time:utc, the x-axis is aligned with the points on the line, but it starts and ends one day before the dataset. It starts on the 17th and ends on the 21st.
![image](https://github.com/user-attachments/assets/3a52a030-fe1a-4e48-8b71-ab8b4200cec8)

**Reply by mbostock** [reply]
How is your data defined? What is the definition of `caloriasPorDia`? If it’s backed by a data file, what does that data look like?

**Reply by PaulaIzzo** [reply]
I was able to do it with type: utc and not indicating tickFormat: d3.timeFormat("%b %d"), this is how the code would look. Thank you very much!
```js
lin_cal = Plot.plot({
  title: "Línea de Tiempo de Calorías Diarias",
  x: {
    label: "Día",
    type: "utc", 
    ticks: 5, 
    //tickFormat: d3.timeFormat("%b %d"), 
  },
  y: { 
    label: "Calorías", 
    grid: true 
  },
  marks: [
    Plot.line(caloriasPorDia, { x: "Día", y: "Calorías", strokeWidth: 3 }),
    Plot.dot(caloriasPorDia, { x: "Día", y: "Calorías", tip: true }),
    Plot.ruleY([1200], { stroke: null })
  ],
  marginBottom: 70,
});
```
![image](https://github.com/user-attachments/assets/a7d4b5be-8271-43fd-978b-0165d20e39ce)

**Reply by mbostock** [reply]
Glad you got it working. Yep, you don’t want to use d3.timeFormat in conjunction with `type: "utc"` because d3.timeFormat uses local time. If you do find you want to specify a custom format for the ticks, you can say `tickFormat: "%b %d"` and Plot will automatically promote it to the correct implementation based on the scale type (which is d3.utcFormat for a UTC scale).
